dirsrvtests/tests/suites/syncrepl_plugin/basic_test.py

```diff
@@ -213,7 +214,9 @@ def get_result(self):
 
     def run(self):
         """Start a sync repl client"""
+        ldap.set_option(ldap.OPT_X_TLS_CACERTFILE, os.path.join(self.inst.get_config_dir(), "ca.crt"))
         ldap_connection = TestSyncer(self.inst.toLDAPURL())
+        ldap_connection.set_option(ldap.OPT_X_TLS_REQUIRE_CERT, ldap.OPT_X_TLS_NEVER)
```

Comment:
Are you sure about NEVER ?  (which mean: bypass CA Certificate verification)
The usual safe way to setup ldap connection over TLS or LDAPS is:
    ld.set_option(ldap.OPT_X_TLS_REQUIRE_CERT,ldap.OPT_X_TLS_DEMAND)
    ld.set_option(ldap.OPT_X_TLS_CACERTFILE, f'{ca.pem}')
    ld.set_option(ldap.OPT_X_TLS_NEWCTX, 0)